### DISCUSSION TITLE
what do i do if a program doesnt have a license

### DISCUSSION THREAD

**Discussion by sastofficial** [microsoft/winget-pkgs#38683] [Q&A] [answered]
?

**Comment by OfficialEsco** [comment, marked as answer]
All software is licensed in one way or another, any closed source software is [Proprietary](https://en.wikipedia.org/wiki/Proprietary_software) which basically means heavily [Copyrighted](https://en.wikipedia.org/wiki/Copyright), Open Source Software without a license is RARE and would be licensed as [License-free](https://en.wikipedia.org/wiki/License-free_software) which would still be considered Proprietary because its work-in-progress.
